pom.xml

```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.flockofseagles</groupId>
+    <groupId>org.flockofseagles.app</groupId>
```

Comment:
Was adding `.app` necessary for it to work correctly?

Reply:
Fixed in 12eca2b